src/utils/fhir.js

```diff
@@ -14,4 +22,40 @@ function generateOperationOutcome(code, msg) {
   };
 }
 
-module.exports = { generateOperationOutcome };
+/**
+ * Helper method to GET resources (and all included references) from a FHIR server
+ *
+ * @param {string} server - the sourse server base url
+ * @param {string} type - the type of the resource
+ * @param {string[]} ids - (optional) list of ids to get
+ * @returns promise of axios data
+ */
+function getResources(server, type, ids = null) {
+  const id_param = ids ? `&_id=${ids.join(',')}` : '';
+  const url = `${server}/${type}?_include=*${id_param}`;
+  const headers = { Authorization: 'Bearer admin' }; // TODO: get an access token
+  return axios
+    .get(url, { headers: headers })
+    .then(response => response.data)
+    .catch(err => console.log(err));
+}
+
+/**
+ * Get all knowledge artifacts (from servers registered in the config
+ * file) and save them. Stores all refrenced resources as well.
+ */
+function refreshKnowledgeArtifacts(db) {
+  const servers = db.select('servers', s => s.type === 'KA');
+  servers.forEach(server => {
+    getResources(server.endpoint, 'PlanDefinition').then(data => {
```

Comment:
One last super minor thing, as you address the lint issues. Can you add a debug log line here that reports the server it's fetching from?